ansible-ipi-install/roles/node-prep/vars/main.yml

```diff
@@ -19,3 +19,7 @@ package_list:
 
 cache_package_list:
   - podman
+
+# Temporary state variables for disconnected registry
+drm_set: false
+dra_set: false
```

Comment:
These are values that can change right? If so they belong in defaults/main.yml . vars/main.yml are for vars that are pretty static that you don't want others modifying.

Reply:
these are not meant for users to change. I put them in vars because of that. These change only when the tasks in the playbook determine they should be true.